flaml/nlp/utils.py

```diff
@@ -12,6 +12,10 @@
 )
 
 import pandas as pd
+import logging
+
+logger = logging.getLogger("flaml.automl")
```

Comment:
why "flaml.automl"?

Reply:
I just saw this line from model.py so just trying to copy it. If it's not flaml.automl, what should it be?